avocado/core/streams.py

```diff
@@ -1,7 +1,16 @@
 #: Builtin special keywords to enable set of logging streams
 BUILTIN_STREAMS = {
+    "app": "avocado.app",
+    "test": "avocado.test",
+    "job": "avocado.job",
+    "debug": "avocado.app.debug",
```

Comment:
Why is there a separate namespace for the app debug level instead of using the `app` namespace with debug level of verbosity?

Reply:
Thanks for spotting this, this has been removed in #5652 which was introduced in parallel with this PR and I have wrongly resolved the collisions. I will fix it.